content/en/docs/architecture/step-registry.md

```diff
@@ -409,86 +427,81 @@ ref:
     limits:
       memory: 4Gi
   env:
-  - name: OPENSHIFT_TEST_SUITE
+  - name: TEST_SUITE
 {{< / highlight >}}
 
-A test or workflow that utilizes this step must give a value to the `OPENSHIFT_TEST_SUITE` parameter, which will be available as an environment
-variable when it is executed. Different tests can be generated by setting different values, which can make generating simple
-variations easier. More complex combinations are encouraged to use separate steps instead.
+Omitting a default value makes `TEST_SUITE` a required parameter. A test that
+wishes to use this step must give the parameter a value in its corresponding
+`env` section. Failing to do so will result in a validation error.
 
-Each parameter declaration in the `env` section consists of the following fields:
-
-* `name`: environment variable name
-* `default` (optional): the value assigned if no other node in the hierarchy
-  provides one (described in the [hierarchical propagation](#hierarchical-propagation)
-  section).
-* `documentation` (optional): a textual description of the parameter
+{{< highlight yaml >}}
+tests:
+- as: e2e
+  steps:
+    test:
+    - ref: openshift-e2e-test
+    env:
+      TEST_SUITE: openshift/conformance/parallel
+{{< / highlight >}}
 
-### Required Parameters
+{{< alert title="Warning" color="warning" >}}
+Parameters must be declared in the `env` section of every step that requires
+them, as shown here: setting values in parent components is not sufficient.
+Basic compliance with this rule is enforced and simple cases of unused values
+will result in validation errors, but not all can be detected, resulting in
+parameter values not being set.
+{{< /alert >}}
 
-Any variable that is not assigned a default value is considered required and must be set at a higher level of the hierarchy. When
-the configuration is resolved, tests that do not satisfy this requirement will generate a validation failure.
+{{< alert title="Warning" color="warning" >}}
+Note that `env` is a top-level field in steps (and chains), alongside the `as`
+field, while it is placed in the `steps` field in tests (and workflows). The
+strict YAML validation used to parse these files will generate an error, but
+this is still a common source of confusion.
```

Comment:
The two alert boxes without any accompanying text look awkward when rendered. The alert boxes are usually used to stress out some important aspect of the base text or clarify a potential pitfall or something like that, but here there's no base text in the fuirst place.